admin_guide/managing_projects.adoc

```diff
@@ -0,0 +1,171 @@
+binding] will deny permissions for self-provisioning any new projects. When
+disabling self-provisioning, set the `projectRequestMessage` parameter in the
+*_master-config.yaml_* file instructing developers on how to request a new
+project. This parameter is a string that will be presented to the developer in
+the web console and command line when they attempt to self-provision a project.
+For example:
+
+----
+Contact your system administrator at projectname@example.com to request a project.
+----
+
+or:
+
+----
+To request a new project, fill out the project request form located at
+https://internal.example.com/openshift-project-request.
+----
+
+[[using-node-selectors]]
+== Using Node Selectors
+
+Node selectors are used in conjunction with labels to control pod placement. 
+
+[NOTE]
+====
+Labels can be assigned link:../install_config/install/advanced_install.html#configuring-node-host-labels[during an advanced installation], or link:../admin_guide/manage_nodes.html#updating-labels-on-nodes[added to a node after installation].
+====
+
+As an OpenShift cluster administrator, you can set the cluster-wide default node selector. This means that you can restrict pod placement to set nodes. For example, if you have an OpenShift topology with multiple regions, node selectors help restrict specific OpenShift projects to only deploy pods onto nodes in a specific region.
+
+OpenShift developers can set a node selector on their project configuration if they wish to restrict nodes even further. This will be added to the cluster-wide node selector, meaning it will be more restrictive.
+
+.Creating a Cluster-wide Node Selector
+
+To create a project with a node selector, use the `--node-selector` option. The following creates a new project named `myproject` dictating that pods be deployed onto nodes labeled `user-node` and `east`:
+
+====
+----
+$ oadm new-project myproject \
+    --node-selector='type=user-node,region=east'
+----
+====
+
+Once this command is run, this becomes the default node selector for all projects.
+
+[NOTE]
+====
+While the `new-project` option is available for both `oadm` and `oc`, the cluster administrator and developer commands respectively, creating a new project with a node selector is only available with the `oadm` command. The `new-project` option is not available to project developers when self-provisioning projects.
+====
+
+Using the above command adds an `annotation` section to the project configuration. Changing the `openshift.io/node-selector` value overrides the default:
+
+====
+----
+...
+metadata:
+  annotations:
+    openshift.io/node-selector: type=user-node,region=east
+...
+----
+====
+
+If the `openshift.io/node-selector` value remains empty, the project will not have a node selector, even if the cluster-wide default has been set. This means that, as an OpenShift administrator, you can set a default for any developers, but want to keep projects separate for infrastructure projects.
```

Comment:
"remains empty" is ambiguous here. There is a difference between not having any annotation (which is the default, leaving the node selector to the cluster default if any), and having one with an empty value "" (which replaces the cluster default with an empty selector). To make it more confusing, there's no way AFAIK to create the latter condition with `oadm new-project`; you have to edit the namespace (not the project!) after creating it. Yuk.

Also the second sentence doesn't read right. Suggestion given all this:

"
If the `openshift.io/node-selector` annotation is given the empty string (`""`), the project will have an empty base node selector, even if the cluster-wide default has been set. This means that, as an OpenShift administrator, you can set a default to restrict developer projects to a subset of nodes and still enable infrastructure projects to schedule the entire cluster.
"